build/travis/job2_AppImage/build.sh

```diff
@@ -27,17 +29,31 @@ else
   makefile_overrides="" # use Makefile defaults
 fi
 
-# Build MuseScore AppImage inside Docker image
-docker run -i -v "${PWD}:/MuseScore" library/centos:6 \
-   /bin/bash -c "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+# Build AppImage. Are we cross-compiling?
+case "$1" in
+  --32bit )
+    shift
+    # Build MuseScore AppImage inside 32-bit Docker image
+    docker run -i -v "${PWD}:/MuseScore" toopher/centos-i386:centos6 /bin/bash -c \
+      "linux32 --32bit i386 /MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+  * )
+    [ "$1" == "--64bit" ] && shift || true
+    # Build MuseScore AppImage inside native (64-bit) Docker image
+    docker run -i -v "${PWD}:/MuseScore" library/centos:6 /bin/bash -c \
+      "/MuseScore/build/Linux+BSD/portable/Recipe $makefile_overrides"
+    ;;
+esac
```

Comment:
Cool.
I had to make some changes to `copy-libs` for this PR so I ended up including all of the [changes we discussed on your branch](https://github.com/ericfont/MuseScore/commit/5a0c4690076d00cd9b0c26330ba1d60e081e29e5), except for the `^processor` change to the Makefile because I felt that belongs in the ARM PR so we know why it's there when reviewing the code a few years from now. You basically just need to make that change and add your `Jessie` (or eqivalent) function to `copy-libs` and update the dependency section of the Recipe for your target distribution.